crypto.c

```diff
@@ -207,62 +207,57 @@ dtls_p_hash(dtls_hashfunc_t h,
 	    const unsigned char *random1, size_t random1len,
 	    const unsigned char *random2, size_t random2len,
 	    unsigned char *buf, size_t buflen) {
-  dtls_hmac_context_t *hmac_a, *hmac_p;
+  dtls_hmac_context_t hmac_a, hmac_p;
```

Comment:
My feeling, it could be done with only one.
Maybe worth a try?

Reply:
See 574f616. Seems to work. At least with the tests I did in RIOT.

Reply:
Sorry just noticed, making this a fixup, does not make sense. Will reword.

Reply:
> See [574f616](https://github.com/eclipse/tinydtls/commit/574f6161b83b20253c8e11194b25eda4717d3bb5).

Now 49de0a5